### DISCUSSION TITLE
Asking for permission to recreate Farmer's Delight as a Vanilla Datapack

### DISCUSSION THREAD

**Discussion by archonxgames** [vectorwing/FarmersDelight#462] [Q&A]
Farmer's Delight has always been one of my top favorite mods, and I'm currently trying to learn how to build vanilla datapacks from scratch. I'm hoping to gain the permission to adapt the code and possibly the assets from Farmer's Delight to create a vanilla-friendly version of the mod. I am not planning to distribute it but I am open to sharing the code and any progress I make if given permission.

**Comment by Omnomfish** [comment]
if you do get permission would you be willing to share it with me? i use a macbook and its hot enough just using notes but running modded minecraft makes it set fires (kidding... mostly. it DOES singe papers though 😅)
